aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -94,6 +94,10 @@ static inline void batchCheckErrors(std::vector<int64_t>& infos, const char* nam
     } else if (info > 0) {
       if (strstr(name, "svd")) {
         AT_ERROR(name, ": the updating process of SBDSDC did not converge (error: ", info, ")");
+      } else if (strstr(name, "eig")) {
+        AT_ERROR(name, ": For batch ", i, ": the QR algorithm failed to compute all the eigenvalues, "
+                 "and no eigenvectors have been computed; elements and ", info+1, ":N of W contain "
```

Comment:
Given that this is raising an error, will the content actually be properly filled with the right values? If it's an out-of-place operation, the user won't even have access to the result.
I think you can either make this a warning with TORCH_WARN stating that some elements are wrong. Or make this an error only stating that the QR algorithm failed for that batch element.
What do you thinkg?